ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/WithErrors.java

```diff
@@ -85,7 +84,6 @@ public Pipeline getPipeline() {
     @Override
     public void finishSpecifyingOutput(String transformName, PInput input,
         PTransform<?, ?> transform) {
-      errors().setCoder(PubsubMessageWithAttributesCoder.of());
```

Comment:
do we just not need this anymore?

Reply:
I think this depends on what type information is available to the particular transform. There is a `PubsubCoderProviderRegistrar` that gets loaded as part of the google cloud beam package. In this particular case of the error collection for `WithErrors`, the collection has a concrete type of `PCollection<PubsubMessage>`, so it's able to use the registered coder.

For MapElementsWithErrors, the output collection type is a type parameter, so the coder registry is not able to infer. Thus, we still do need to explicitly set coder when the output collection type is pubsubmessage.

Reply:
ah, makes sense. thanks for the explanation!